### PR TITLE
added new CreateConnection class

### PR DESCRIPTION
**Pull request by M4jo**
Added new class that creates a connection to a remote server

### CODE REVIEW

File: src/web_backups/main/ui/createConnection/CreateConnection.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            Session session = jsch.getSession(username, host, port);
+            session.setPassword(password);
+            session.setConfig("StrictHostKeyChecking", "no");
+            session.setTimeout(15000);
+            session.connect();
+        } catch (JSchException e) {
+            e.printStackTrace();
+        }
+    }
+    public static void main(String[] args) {
```
Comment (arti1221) [inline src/web_backups/main/ui/createConnection/CreateConnection.java]: This should be removed as the application is being started from a different class.

File: src/web_backups/main/ui/createConnection/CreateConnection.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        String username = "webbackup";
```
Comment (arti1221) [inline src/web_backups/main/ui/createConnection/CreateConnection.java:22]: All attributes should be declared via constructor. Please make sure to make these attributes private.

File: src/web_backups/main/ui/createConnection/CreateConnection.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
+package web_backups.main.ui.createConnection;
```
Comment (arti1221) [inline src/web_backups/main/ui/createConnection/CreateConnection.java]: This should be moved to a different package as it is not a UI functionality but making a connection to site specified in config. file. Please move this into global library.

File: src/web_backups/main/ui/createConnection/CreateConnection.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+public class CreateConnection {
+    public static void connect(String username, String host,int port, String password) {
+        try {
+            JSch jsch = new JSch();
+            Session session = jsch.getSession(username, host, port);
+            session.setPassword(password);
+            session.setConfig("StrictHostKeyChecking", "no");
+            session.setTimeout(15000);
+            session.connect();
+        } catch (JSchException e) {
```
Comment (arti1221) [inline src/web_backups/main/ui/createConnection/CreateConnection.java]: Please add a validation message for the user. Use NoValidDataException.java instead.

File: src/web_backups/main/ui/createConnection/CreateConnection.java (hunk 3 above)
Comment (arti1221) [inline src/web_backups/main/ui/createConnection/CreateConnection.java]: Please add Jsch package to libraries as it is not a default java package.

**Review by arti1221 — CHANGES_REQUESTED** [review]
Please remove pvsm

**Review by yarvelian — APPROVED** [review]
Nice

**Pull request closed without merging**